src/lug/Window/Unix/WindowImplX11.cpp

```diff
@@ -0,0 +1,16 @@
+#include <lug/Window/Unix/WindowImplX11.hpp>
+
+lug::Window::priv::WindowImpl::WindowImpl()
+{
+}
+
+lug::Window::priv::WindowImpl::~WindowImpl()
+{
+}
+
+
+bool lug::Window::priv::WindowImpl::create(int width, int height, const std::string& title, uint32_t style)
```

Comment:
I believe uint16_t is more appropriate? Can window size really exceed 65,535 by 65,535? (honest question here).